zerver/lib/push_notifications.py

```diff
@@ -144,13 +144,21 @@ class APNsContext:
     loop: asyncio.AbstractEventLoop
 
 
+def apns_enabled() -> bool:
+    if settings.APNS_TOKEN_KEY_FILE is not None:
+        return True
+    if settings.APNS_CERT_FILE is not None:
+        return True
+    return False
```

Comment:
Applied this tweak to work around this causing a coverage failure:

``` diff
diff --git a/zerver/lib/push_notifications.py b/zerver/lib/push_notifications.py
index 3804508fe1..9f7acee4b4 100644
--- a/zerver/lib/push_notifications.py
+++ b/zerver/lib/push_notifications.py
@@ -146,11 +146,7 @@ class APNsContext:
 
 
 def apns_enabled() -> bool:
-    if settings.APNS_TOKEN_KEY_FILE is not None:
-        return True
-    if settings.APNS_CERT_FILE is not None:
-        return True
-    return False
+    return settings.APNS_TOKEN_KEY_FILE is not None or settings.APNS_CERT_FILE is not None
 
 
 @lru_cache(maxsize=None)
```

Reply:
Ha, OK then.